### PR TITLE
salkvus has finished Chapter1

### PR DESCRIPTION
**Pull request by salkvus**
### Solutions for Chapter 1

cc @vrom911 @chshersh

### CODE REVIEW

File: src/Chapter1.hs [hunk 1]
```diff
@@ -568,7 +573,10 @@ True
 >>> isVowel 'x'
 False
 -}
-isVowel c = error "isVowel: not implemented!"
+isVowel :: Char -> Bool
+isVowel c
+        | c == 'a' || c == 'e' || c == 'i' || c == 'o' || c == 'u' = True
+        | otherwise = False
```
Comment (vrom911) [inline src/Chapter1.hs:579]: Nice one! 👍🏼 
As `==` returns `Bool` itself, you can just return the result of this function:

```haskell
isVowel c = c == 'a' || c == 'e' || c == 'i' ...
```

File: src/Chapter1.hs [hunk 2]
```diff
@@ -628,11 +636,17 @@ Implement a function that returns the sum of the last two digits of a number.
...
+
 Try to introduce variables in this task (either with let-in or where) to avoid
 specifying complex expressions.
 -}
 
-sumLast2 n = error "sumLast2: Not implemented!"
+sumLast2 :: Int -> Int
+sumLast2 n =
+      let num = mod (abs n) 100
+          in div num 10 + mod num 10
```
Comment (vrom911) [inline src/Chapter1.hs:649]: That is a wonderful solution! 👏🏼  You correctly noticed that it is the `div` and `mod`, cool 😎 

One hint to make your solution even shorter: you can see that you use both:

```haskell
mod num 10
div num 10
```
The standard library has the [`divMod`](https://hackage.haskell.org/package/base-4.14.0.0/docs/Prelude.html#v:divMod) function, that actually combines inside both `div` and `mod`. And this is exactly what you use!.

So you could write it this way:

```haskell
(x, y) = divMod num 10
```

You can see how we could pattern match on the pair 🙂

**Review by vrom911 — APPROVED** [review]
Awesome work 👍🏼

File: src/Chapter1.hs (hunk 2 above)
Comment (salkvus) [reply src/Chapter1.hs:649]: Thank you for your helpful comments

File: src/Chapter1.hs (hunk 1 above)
Comment (salkvus) [reply src/Chapter1.hs:579]: It turns out to be so simple :) Thanks

**Comment by chshersh** [comment]
We can't merge solutions to our repository, so I'm closing this PR.
Feel free to open PRs to unsolved chapters in your fork of this repository 🍴

**Pull request closed without merging**